ccmlib/scylla_node.py

```diff
@@ -187,13 +187,11 @@ def _start_jmx(self, data):
                 '-jar',
                 jmx_jar]
         log_file = os.path.join(self.get_path(), 'logs', 'system.log.jmx')
-        jmx_log = open(log_file, 'a')
         env_copy = os.environ
         env_copy['SCYLLA_HOME'] = self.get_path()
-        self._process_jmx = subprocess.Popen(args, stdout=jmx_log,
-                                             stderr=jmx_log,
-                                             close_fds=True,
-                                             env=env_copy)
+
+        with open(log_file, 'a') as jmx_log:
```

Comment:
doesn't this mean or log file would be closed, right after this call ?
did you checked the the logs of jmx, and scylla are properly written when this context manager is used like that ?

Reply:
FYI, `Popen(close_fds=True)` already close those files, regardless of the context manager.

Reply:
in cases of errors, yep it might be left open